src/Badge/__tests__/__snapshots__/Badge.test.js.snap

```diff
@@ -47,7 +46,7 @@ exports[`<Badge /> Colors properly renders a blue badge 1`] = `
 
 <div
   className="emotion-4 emotion-5"
-  color="blue"
+  color=""
```

Comment:
Should all of these be empty strings? 🤔

Reply:
Good call! This was the result of the defaultProp in `StyledBadge` not being updated to `paletteColor`